atomic_reactor/plugins/pre_fetch_maven_artifacts.py

```diff
@@ -114,7 +114,9 @@ def read_nvr_requests(self):
             self.log.debug('%s not found', self.NVR_REQUESTS_FILENAME)
             return []
 
-        nvr_requests = util.read_yaml(file_path, 'schemas/fetch-artifacts-nvr.json')
+        with open(file_path) as f:
+            yaml_data = f.read()
+        nvr_requests = util.read_yaml(yaml_data, 'schemas/fetch-artifacts-nvr.json')
```

Comment:
@twaugh  what do you think? you wanted to open() in callers

Reply:
I wanted to avoid overloading the meaning of a parameter. Adding a wrapper function seems like a good way forward. (As I said yesterday, the other approach would be to have two optional keyword parameters for the function, one for a filename and one for a file-like object.)

Reply:
ok